cmd/nerdctl/system_prune_linux_test.go

```diff
@@ -51,4 +61,39 @@ func TestSystemPrune(t *testing.T) {
 	base.Cmd("ps", "-a").AssertNoOut(tID)
 	base.Cmd("network", "ls").AssertNoOut(nID)
 	base.Cmd("images").AssertNoOut("alpine")
+
+	if rootlessutil.IsRootless() {
+		t.Skip("test skipped for buildkitd is not available with rootless containers")
```

Comment:
Doesn't seem true

Reply:
https://github.com/containerd/nerdctl/pull/1360#discussion_r984446139

I remember that calling `buildctl` would be failed in the rootless container integration tests. So I append these lines of code.

Reply:
What is the error on rootless?

Reply:
> I remember that calling buildctl would be failed in the rootless container integration tests. 

I could not reproduce that error on rootless container integration tests with my local machine now. 🙈

I would remove these lines.